agent/drive_handler.go

```diff
@@ -146,3 +164,87 @@ func isStubDrive(d drive) bool {
 
 	return internal.IsStubDrive(f)
 }
+
+func (dh driveHandler) MountDrive(ctx context.Context, req *drivemount.MountDriveRequest) (*empty.Empty, error) {
+	logger := log.G(ctx).WithField("MountDriveRequest", req.String())
```

Comment:
`MountDriveRequest` has lots of fields. I dont think we want to repeat them in every log message below. Would it make sense to log those fields once in `Debugf`?

Reply:
Yeah good point, will do